docs/metrics.md

```diff
@@ -96,6 +96,10 @@ Histogram record the [distribution](http://kamon.io/documentation/0.6.x/kamon-co
 * `my_metric.max` - Max of aggregated values during the flush interval.
 * `my_metric.min` - Min of aggregated values during the flush interval.
 
+#### Gauges
+
+Gauges record the [distribution](http://kamon.io/documentation/0.6.x/kamon-core/metrics/instruments/#gauges) of given metric and there names are prefixed with `openwhisk.gauge`. For example `openwhisk.gauge.loadbalancer_totalHealthyInvoker_count`. A gauge metrics provides the value at the given point and reports the same data unless the value has been changed be incremental or decremental than before. Gauges are useful for reporting metrics like kafka queue size or disk size.
```

Comment:
Better to use latest docs link https://kamon.io/docs/latest/core/metrics/

Reply:
I will update